### DISCUSSION TITLE
Why links are reloading current page after setting up custom domain?

### DISCUSSION THREAD

**Discussion by sysadev** [Knogobert/ntn-boilerplate#12] [Q&A]
With netlify subdomain I'm having no issue but when I setup custom domain links started to reload the current page instead of visiting the clicked link.

[netlify subdomain](https://sysa.netlify.app/)
[custom domain](https://shoaiybsysa.ml/)

**Comment by Knogobert** [comment]
It seems to me that the netlify link suffers from the same issue. It is most likely something that has changed in your code that causes this, as https://ntn-boilerplate.netlify.app/ still works as intended. Please check out your setup a bit closer, my guess is that it might be caused in `nuxt.config.js`. Good luck!
